Overview.bs

```diff
@@ -1515,39 +1491,41 @@ result in an extended [=font subset=]:
     space that covers at least the union of the axis space the client has and the axis space the
     client needs.</span>
 
-If the checksum of the [=original font=] computed by the procedure in [[#computing-checksums]] does not
-match the checksum in [=PatchRequest/original_font_checksum=] or
-[=PatchRequest/original_font_checksum=] is unset, then:
 
-*  <span class="conform server" id="conform-response-original-checksum">The value of
-    [=PatchResponse/original_font_checksum=] must be set to the checksum of the [=original font=]
-    computed by the procedure in [[#computing-checksums]].</span>
+*  <span class="conform server" id="conform-response-client-state">That has a
+    <a href="https://learn.microsoft.com/en-us/typography/opentype/spec/otff#table-directory">table</a>
+    which is mapped to the tag "IFTP" whose content is a single [=ClientState=] object encoded via
+    CBOR:</span>
```

Comment:
Suggested wording change: replace *table which is mapped to the tag "IFTP"* to *table identified by the tag 'IFTP'* (also note the use of single quotation marks here and elsewhere for table tag definitions).